check_borg

```diff
@@ -84,21 +95,21 @@ sec_crit="$(date --date="${crit}" '+%s')"
 
 # check warning and critical values
 if check_range "${sec_crit}" 0:"${sec_warn}" ; then
-	# 
+	#
 	error "Warning value has to be a more recent timepoint than critical."
 fi
 
 # get unixtime of last backup
-sec_last="$(borg list --format '{time}{NUL}' "${repo}" | xargs '-0' '-I&' 'date' '--date=&' '+%s' | sort | tail -n 1)"
-last="$(date --date="@${sec_last}")"
+last="$(borg list --format '{time}' --sort-by timestamp --last 1 ${repo})"
```

Comment:
- this is only compatible with borg 1.1 (`--sort-by` is missing in 1.0). Please assure, that we run borg 1.1 first and if not exit with state unknown or use a fallback.
- please quote the `${repo}` variable.

Reply:
ok, `--last 1` is also missing in 1.1. It doesn't make sense to add conditional code to simplify the call, so I'll just remove this commit, too.

Reply:
@zauberpony Well, both options had been added in borg 1.1. Your code is much cleaner and I'd like to have your code. IMO it's ok to require borg 1.1 now.

Could you please add the stuff **later** on a **new** pull request? (So I can tag a release for the bugfix you made without requiring borg 1.1).